js/toc.js

```diff
@@ -0,0 +1,98 @@
+// https://github.com/ghiculescu/jekyll-table-of-contents
```

Comment:
Can you point the serving version to the one in third_party/jekyll-table-of-contents/, rather than this copy?